packages/firebase-frameworks/README.md

```diff
@@ -0,0 +1,135 @@
+# Firebase CLI & Web Frameworks
+
+## Frameworks
+
+| Framework  | Support | |
+| ---------- | ------- | - |
+| [Next.js](https://firebase.google.com/docs/hosting/frameworks/nextjs) | **Early preview** | |
```

Comment:
nit* suggest italics, since bold conflicts with table headers.

Reply:
oh theses are from old doc. feel free to just copy and paste.

Reply:
yeah this whole section is just the old docs copied in since they're still relevant for this section and James and I figured it might make sense to keep a copy of them for now